launcher/game/translations.rpy

```diff
@@ -27,6 +27,9 @@ init python:
     if persistent.translate_language is None:
         persistent.translate_language = "english"
 
+    if persistent.extract_language is None:
```

Comment:
This should probably be replaced with `persistent.extract_language = persistent.extract_language or "None"`.
That way, if you simply remove all characters in the input and end up with "" (the empty string), it also gets replaces with "None".
I also think that replacement should not only happen at init time, which is on launcher boot and reload, but every time it may come to use, which means doing it inside the extract_dialogue label.